src/genie/libs/parser/linux/tests/test_route.py

```diff
@@ -0,0 +1,197 @@
+import re
+import unittest
+from unittest.mock import Mock
+
+from pyats.topology import Device
+
+from genie.metaparser.util.exceptions import (SchemaMissingKeyError,
+                                              SchemaEmptyParserError)
+
+from genie.libs.parser.linux.route import Route,\
+                                          ShowNetworkStatusRoute
+
+
+#############################################################################
+# unitest For route
+#############################################################################
+
+class TestRoute(unittest.TestCase):
+    '''
+    Unit test for
+        * route
+    '''
+    
+    device = Device(name='aDevice')
+    maxDiff = None
+    empty_output = {'execute.return_value': ''}
+
+    golden_parsed_output = {
+        '10.10.0.0': {
+            'destination': '10.10.0.0',
+            'flags': 'U',
+            'gateway': '0.0.0.0',
+            'interface': 'eth1-05',
+            'mask': '255.255.255.0',
+            'metric': 0,
+            'ref': 0,
+            'use': 0
+                },
+        '172.17.0.0': {
+            'destination': '172.17.0.0',
+            'flags': 'U',
+            'gateway': '0.0.0.0',
+            'interface': 'docker0',
+            'mask': '255.255.0.0',
+            'metric': 0,
+            'ref': 0,
+            'use': 0
+            },
+        '192.168.1.0': {
+            'destination': '192.168.1.0',
+            'flags': 'U',
+            'gateway': '0.0.0.0',
+            'interface': 'wlo1',
+            'mask': '255.255.255.0',
+            'metric': 600,
+            'ref': 0,
+            'use': 0
+            },
+        '192.168.122.0': {
+            'destination': '192.168.122.0',
+            'flags': 'U',
+            'gateway': '0.0.0.0',
+            'interface': 'virbr0',
+            'mask': '255.255.255.0',
+            'metric': 0,
+            'ref': 0,
+            'use': 0
+            },
+        'default': {
```

Comment:
ideal schema would be like below. there is the possibility to see same destination with different mask/interface/gateway. 
```
'routes': {
    'default': {
        'mask': {
            '0.0.0.0': {
                'nexthop': {
                    1: { # index
                        'interface': 'wlo1',
                        'flags': 'UG',
                        'gateway': '_gateway',
                        'metric': 600,
                        'ref': 0,
                        'use': 0,
                    }
                }
            }
        }
    }
}